spotlight-client/src/VizHistoricalPopulationBreakdown/VizHistoricalPopulationBreakdown.test.tsx

```diff
@@ -79,7 +84,52 @@ test("renders total", async () => {
   ).toBeVisible();
 });
 
-// TODO(#278): There should be not less than 240 points
-test.todo("fills in missing data");
```

Comment:
Did this one get lost? Or is this the new comment at the bottom of the file?

Reply:
it's addressed in the metric tests now instead